src/components/Text/Text.jsx

```diff
@@ -1,16 +1,39 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import cn from 'classnames';
+// @flow
+/* eslint react/prop-types: 0 */
+import * as React from 'react';
```

Comment:
Why are we importing all React?

Reply:
for flow https://flow.org/en/docs/react/children/